Firestore/core/src/firebase/firestore/model/field_value.h

```diff
@@ -146,11 +146,6 @@ class FieldValue : public util::Comparable<FieldValue> {
     return *blob_value_;
   }
 
-  const Map& object_value() const {
```

Comment:
What's the motivation behind avoiding this? All the other platforms define `value` for the object type and this doesn't seem harmful.

Reply:
Only because it's not really necessary. Ideally, if you want to get add the underlying map, you should already have a ObjectValue. (This is true of all the cases so far.) If it becomes necessary, I have no objection to adding it back in.